File: app/model/feedback/EvaluatedAspect.java
package model.feedback;

import model.StringConsts$;

public enum EvaluatedAspect {
  
  // @formatter:off
  USED("Wie oft haben Sie dieses Tool genutzt?", "Oft", "Manchmal", "Selten"),
  
  SENSE("Finden Sie dieses Tool sinnvoll?", "Sinnvoll", StringConsts$.MODULE$.NEUTRAL(), "Sinnlos"),
  
  USABILITY("Wie bewerten Sie die allgemeine Bedienbarkeit dieses Tools?", "Gut", StringConsts$.MODULE$.NEUTRAL(), "Schlecht"),  
  
  STYLE_OF_FEEDBACK("Wie bewerten Sie die Gestaltung des Feedbacks dieses Tools?", "Gut", StringConsts$.MODULE$.NEUTRAL(), "Schlecht"),
  
  FAIRNESS_OF_FEEDBACK("Wie bewerten Sie die Fairness der Evaluation dieses Tools?", "Fair", StringConsts$.MODULE$.NEUTRAL(), "Unfair");
  // @formatter:on
  
  private String question;
  private String positive;
  private String neutral;
  private String negative;
  
  private EvaluatedAspect(String theQuestion, String thePositive, String theNeutral, String theNegative) {
    question = theQuestion;
    positive = thePositive;
    neutral = theNeutral;
    negative = theNegative;
  }
  
  public String getQuestion() {
    return question;
  }
  
  public String getPositive() {
    return positive;
  }
  
  public String getNeutral() {
    return neutral;
  }
  
  public String getNegative() {
    return negative;
  }
  
}

File: server/app/model/feedback/EvaluatedAspect.scala
package model.feedback

import enumeratum.{EnumEntry, PlayEnum}

sealed abstract class EvaluatedAspect(
  val question: String,
  val positive: String = "Gut",
  val neutral: String = "Neutral",
  val negative: String = "Schlecht"
) extends EnumEntry


object EvaluatedAspects extends PlayEnum[EvaluatedAspect] {

  override def values: IndexedSeq[EvaluatedAspect] = findValues


  case object USED extends EvaluatedAspect("Wie oft haben Sie dieses Tool genutzt?", positive = "Oft", "Manchmal", negative = "Selten")

  case object SENSE extends EvaluatedAspect("Finden Sie dieses Tool sinnvoll?", positive = "Sinnvoll", negative = "Sinnlos")

  case object USABILITY extends EvaluatedAspect("Wie bewerten Sie die allgemeine Bedienbarkeit dieses Tools?")

  case object STYLE_OF_FEEDBACK extends EvaluatedAspect("Wie bewerten Sie die Gestaltung des Feedbacks dieses Tools?")

  case object FAIRNESS_OF_FEEDBACK extends EvaluatedAspect("Wie bewerten Sie die Fairness der Evaluation dieses Tools?", positive = "Fair", negative = "Unfair")

}
